index.js

```diff
@@ -1213,23 +1213,23 @@ NodeID3.prototype.createChapterFrameHelper = function(chapter, id) {
 
     let elementIDBuffer = Buffer.from(chapter.elementID + "\0")
     let startTimeBuffer = Buffer.alloc(4)
-    startTimeBuffer.writeUInt32BE(chapter.startTimeMs)
+    startTimeBuffer.writeUInt32BE(chapter.startTimeMs, 0)

```

Comment:
These arguments [are required according to the docs](https://nodejs.org/docs/latest-v10.x/api/buffer.html).

In fact the code was not failing in runtime in neither 8 nor 10.